### PR TITLE
Rework `git elegant configure` command

### PR DESCRIPTION
**Pull request by alexbeatnik**
Issue #130 implemented   
    1. git elegant configure renamed to configure-repository
    2. command git configure-repository works like git elegant configure --local
    3. Tests and README updated

### CODE REVIEW

File: src/main/git-elegant-configure-repository [hunk 1]
```diff
@@ -23,8 +23,7 @@ _alias_message="add git aliases for all 'elegant git' commands"
 
 
 _configure() {
-    MODE=$1; shift
-    FUNCTIONS=$@
+ FUNCTIONS=$@
```
Comment (extsoft) [inline src/main/git-elegant-configure-repository]:
```suggestion
     FUNCTIONS=$@
```

File: src/main/git-elegant-configure-repository [hunk 2]
```diff
@@ -42,25 +41,20 @@ _configure() {
...
-GLOBALS=(_user_name _user_email _core_comment_char _apply_whitespace _alias)
-LOCALS=(_user_name _user_email _core_comment_char _apply_whitespace)
-
 default() {
-    case "$1" in
-        --global)   _configure --global ${GLOBALS[@]} ;;
-        --local)    _configure --local ${LOCALS[@]}  ;;
-        *) exit 11 # @todo #9 Add usage message if mandatory argument is not set for `git-elegant configure`.
-    esac
+
```
Comment (extsoft) [inline src/main/git-elegant-configure-repository]: @alexbeatnik please review empty lines. They don't give any value.

File: src/test/git-elegant-configure-repository.bats [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    fake-pass git "config --local apply.whitespace fix"
+}
+
+teardown() {
+    clean-fake
+    clean-git
+}
+
+@test "'configure': '--local' option is available" {
```
Comment (extsoft) [inline src/test/git-elegant-configure-repository.bats]: @alexbeatnik this option is unavailable. WHy do we have this test?

Comment (alexbeatnik) [reply src/test/git-elegant-configure-repository.bats]: @extsoft Thanks for review, fixed

File: src/main/git-elegant-configure-repository [hunk 4]
```diff
@@ -42,25 +41,18 @@ _configure() {
...
-
 default() {
-    case "$1" in
-        --global)   _configure --global ${GLOBALS[@]} ;;
-        --local)    _configure --local ${LOCALS[@]}  ;;
-        *) exit 11 # @todo #9 Add usage message if mandatory argument is not set for `git-elegant configure`.
-    esac
-}
+      _configure _user_name _user_email _core_comment_char _apply_whitespace
+}
```
Comment (extsoft) [inline src/main/git-elegant-configure-repository]: @alexbeatnik please add an empty line to the end of the file.

Comment (alexbeatnik) [reply src/main/git-elegant-configure-repository]: @extsoft Fixed

**Comment by extsoft** [comment]
@rultor please merge.

**Pull request merged**